test/ranking.spec.js

```diff
@@ -115,11 +115,13 @@ describe("Ranking's logic", () => {
     expect(JSON.parse(localStorage.getItem('starships'))).toEqual(results);
   });
 
-  it('Ranking saved in local storage should be returned', () => {
+  it('Ranking saved in local storage should be returned in order', () => {
     const starshipsRanking = new Ranking('starships');
     localStorage.setItem('starships', JSON.stringify(results));
 
-    expect(starshipsRanking.getScores()).toEqual(results);
+    expect(starshipsRanking.getScores()).toEqual(
+      results.sort((a, b) => a.score / a.maxScore - b.score / b.maxScore),
```

Comment:
Tu bym jednak wstawiła zadeklarowaną wcześniej tablicę, żeby mieć pewność, że jest ok. Teraz sprawdzasz tylko czy to, co ta funkcja zwraca jest równe temu, co zwraca ta sama funkcja ;)

Reply:
Fakt, poprawię później :-)

Reply:
@lamparina Bardzo dobra uwaga Ania!
Widzę, że ja nie muszę tego sprawdzać :) 
Dokładnie to samo bym napisał.

Reply:
noooo i wyszło, że coś nie działa sortowanie :-p

Reply:
@lamparina teraz śmiga :-) https://github.com/nowakprojects/CodersCamp2020.Project.JavaScript.StarWarsQuiz/pull/92/commits/6cf882be6a2ffb38bfbbe74188721ca0ff660bab
mergujemy?